requirements.txt

```diff
@@ -4,4 +4,7 @@
 
 oslo.config>=1.4.0  # Apache-2.0
 sqlalchemy
-pyOpenSSL>=0.13.0,<=0.15.1
\ No newline at end of file
+pyOpenSSL>=0.13.0,<=0.15.1
+Click>=5.1
+requests
+python-neutronclient<2.5.0,>=2.4.0
```

Comment:
It doesn't really belong to the apic-ml2 driver either (apic related commands are in the CLI as well).
Would it make sense to have a completely separated project for this?